binding/form_mapping.go

```diff
@@ -38,8 +48,13 @@ func mapForm(ptr interface{}, form map[string][]string) error {
 			}
 		}
 		inputValue, exists := form[inputFieldName]
+
 		if !exists {
```

Comment:
if inputValue is empty array(inputField exist and its ""), default value not take effect

Reply:
can you post one example? thanks!